managed-identity-keyvault/README.md

```diff
@@ -0,0 +1,50 @@
+# Access Key Vault with managed identity
+
+This sample shows how to access Key Vault with managed identity in `Azure Spring Cloud`.
+
+## Prerequisite
+
+* [JDK 8](https://docs.microsoft.com/en-us/azure/java/jdk/java-jdk-install)
+* [Maven 3.0 and above](http://maven.apache.org/install.html)
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) or [Azure Cloud Shell](https://docs.microsoft.com/en-us/azure/cloud-shell/overview)
+* An existing Key Vault. If you need to create a Key Vault, you can use the Azure Portal or [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest).
```

Comment:
Let's put a link to the official Key Vault docs: https://docs.microsoft.com/en-us/azure/key-vault/secrets/quick-create-cli#create-a-key-vault

Reply:
Use CLI docs (https://docs.microsoft.com/en-us/cli/azure/keyvault?view=azure-cli-latest#az-keyvault-create) instead? since on this page, customers can find other commands about spring-cloud/rbac easily?

Reply:
Sounds good.